keystrokes/src/components/Character/CharacterDetails/CharacterDetailPage.js

```diff
@@ -70,16 +74,23 @@ function CharacterDetailPage({ userId, match }) {
 				updates[`/Keybindings/${key}`] = allKeybindings[key];
 			});
 		}
-
-		const error = await ref.update(updates);
-		setAlertMessage(error);
+		try {
+			const error = await ref.update(updates);
```

Comment:
Is there always an error one update or if there isn't one will it just be empty?

Reply:
yea itll just be null if it there isnt an error